### PR TITLE
Fix for #729 - Removed Pre-defined profile properties

### PR DESCRIPTION
**Pull request by Shruthi-1MN**
**What this PR does / why we need it**:
This is to fix https://github.com/opensds/opensds/issues/729

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
Profile for file share has some pre-defined properties

**Special notes for your reviewer**:
Sample commands to execute and verify the fix
```
root@shruthi-VirtualBox:~/go/src/github.com/opensds/opensds# curl --header "Content-Type: application/json" --data '{"name":"f12", "description":"file share test", "storageType":"file"}' http://127.0.0.1:50040/v1beta/e93b4c0934da416eb9c8d120c5d04d96/profiles
{"id":"e372f004-b2a9-4ffe-8903-a0f6a11563ee","createdAt":"2019-06-13T12:04:25","updatedAt":"","name":"f12","description":"file share test","storageType":"file","provisioningProperties":{"dataStorage":{"isSpaceEfficient":false},"ioConnectivity":{}},"replicationProperties":{"dataProtection":{"isIsolated":false},"replicaInfos":{}},"snapshotProperties":{"schedule":{},"retention":{},"topology":{}},"dataProtectionProperties":{"dataProtection":{"isIsolated":false}}}
root@shruthi-VirtualBox:~/go/src/github.com/opensds/opensds# osdsctl profile list
WARNING: OPENSDS_ENDPOINT is not specified, use default(http://localhost:50040)
WARNING: Not found Env OPENSDS_AUTH_STRATEGY, use default(noauth)
+--------------------------------------+---------+-----------------+-------------+
| Id                                   | Name    | Description     | StorageType |
+--------------------------------------+---------+-----------------+-------------+
| ecd0fe58-4a1d-445d-924c-deb9a991c646 | default | default policy  | file        |
| e372f004-b2a9-4ffe-8903-a0f6a11563ee | f12     | file share test | file        |
| 999977f2-5a52-4f64-9cc5-7d6df6685993 | default | default policy  | block       |
+--------------------------------------+---------+-----------------+-------------+
```

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by leonwanghui** [comment]
I'm wondering why do you remove the code? Please elaborate.

File: pkg/api/controllers/profile.go [hunk 1]
```diff
@@ -60,37 +59,13 @@ func (p *ProfilePortal) CreateProfile() {
...
-			}
-		}
 		break
 	default:
 		errMsg := fmt.Sprintf("parse profile request body failed: %v is invalid storagetype", stype)
 		p.ErrorHandle(model.ErrorBadRequest, errMsg)
 		return
 	}
-
+
```
Comment (leonwanghui) [inline pkg/api/controllers/profile.go]: Please remove the blank spaces.

Comment (Shruthi-1MN) [reply pkg/api/controllers/profile.go]: Removed

**Comment by Shruthi-1MN** [comment]
> I'm wondering why do you remove the code? Please elaborate.

It is optional parameter now, initially we decided to use as default fileshare storage access capability with some default values, but later decided to add these capability in later releases.

**Pull request merged**